packages/subgraph/tests/utils/utils.test.ts

```diff
@@ -0,0 +1,36 @@
+import {BigInt} from '@graphprotocol/graph-ts';
+
+import {assert, describe, test} from 'matchstick-as/assembly/index';
+import {bigIntToBytes32} from '../../src/utils/bytes';
+import {
+  ZERO,
+  ONE,
+  TWO,
+  MAX_UINT256_NUMBER_STRING,
+  ZERO_BYTES32,
+  ONE_BYTES32,
+  HALF_UINT256_BYTES32,
+  MAX_UINT256_BYTES32
+} from '../constants';
+import {TestCase} from './utils';
+
+describe('Test bytes', function() {
+  test('`bigIntToBytes32` with a range of `bigInt`s', function() {
+    const MAX_UINT256 = BigInt.fromString(MAX_UINT256_NUMBER_STRING);
+
+    const testCases: TestCase[] = [
```

Comment:
about the `TestCase` ? do we have alternative ? This could be an overkill.

1. Would love to see if it's a recommended practice and in the long run, it won't. hurt us instead of helping us.
2. Depending on answer 1, if we have to have it, then `TestCase` might be a wrong name. What if I wanna create another TestCase which instead of `BigInt`, expects `String` as an argument. Thats why you will need to be specific to name it `TestCaseInputBigInt` and this is where It might start to become an overkill.